py3status/modules/thunderbird_calendar.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+"""
+Display tasks in thunderbird calendar.
+
+Configuration parameters:
+    - cache_timeout : how often we refresh usage in seconds (default: 120s)
+    - profile_path : path to the user thunderbird profile (not optional)
+    - format : see placeholders below
```

Comment:
Could you change `- cache_timeout : ` to `cache_timeout:`

This is for the new documentation.  I really should write a guide to the docstrings explaining this